src/modules/map/components/LocationPopup.vue

```diff
@@ -89,7 +92,7 @@ watch(clickInfo, (newClickInfo) => {
     if (newClickInfo) {
         updateWhat3Word()
         updateHeight()
-        updateShareLink()
+        showEmbedSharing.value = false
     }
 })
 watch(currentLang, () => {
```

Comment:
Here the share link should only be updated if the embedsharing is shown

Reply:
What is the intended behaviour here?
Should the share button upon opening a new popup window close or should it stay open and generate a new shortlink?